Mathlib/Algebra/Order/Group/Pointwise/Interval.lean

```diff
@@ -735,108 +837,16 @@ theorem image_const_mul_uIcc (a b c : α) : (a * ·) '' [[b, c]] = [[a * b, a *
 theorem image_div_const_uIcc (a b c : α) : (fun x => x / a) '' [[b, c]] = [[b / a, c / a]] := by
```

Comment:
Does this not also generalize?

Reply:
At least, not without modifying the proofs.